spec/soda_machine_spec.rb

```diff
@@ -51,4 +54,23 @@
     end
   end
 
+  describe "#find_cheapest_soda", { current_inventory_count: true } do
+    it "returns the cheapest soda item in the inventory" do
+      expect(soda_machine.find_cheapest_soda).to eq([second_pepsi, pepsi])
+    end
+
+    context "Sell some soda and find cheapers" do
+      before(:each) do 
+        soda_machine.sell('Mountain Dew')
+        soda_machine.sell('Coke Zero')
+      end
+
+      it "Find the cheapest soda after selling some items" do 
+        expect(soda_machine.find_cheapest_soda).to eq([ second_pepsi, pepsi])
```

Comment:
these are the same as the "cheapest" above although the way the test is written it would make you think that you should find some different cheapest sodas after selling a couple of them.  it's confusing to see this test with the same results -- i expect it to be different.